cmd/clusterctl/cmd/move.go

```diff
@@ -78,6 +81,7 @@ func runMove() error {
 		FromKubeconfig: client.Kubeconfig{Path: mo.fromKubeconfig, Context: mo.fromKubeconfigContext},
 		ToKubeconfig:   client.Kubeconfig{Path: mo.toKubeconfig, Context: mo.toKubeconfigContext},
 		Namespace:      mo.namespace,
+		DryRun:         mo.dryRun,
```

Comment:
In case of dry run, is it possible to tolerate the `toKubeconfig` to be empty?

Reply:
I need more check here, good suggestion

Reply:
updated as well, seems works on my env

```
# ../clusterctl move  -v 10  --dry-run
No default config file available
Performing move...
********************************************************
This is a dryrun mover, will not perform any real action
********************************************************

```